src/libs/ReportUtils.js

```diff
@@ -2200,7 +2200,7 @@ function buildOptimisticIOUReport(payeeAccountID, payerAccountID, total, chatRep
 
         // We don't translate reportName because the server response is always in English
         reportName: `${payerEmail} owes ${formattedTotal}`,
-        notificationPreference: CONST.REPORT.NOTIFICATION_PREFERENCE.ALWAYS,
+        notificationPreference: CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN,
```

Comment:
Could you also add this to the automated IOU tests?

Reply:
I've checked, no one of those edited functions exist in `ReportUtilsTest.js`. So, should I create tests for these functions in a scope of this PR?

Reply:
There are tests with IOU.requestMoney which uses this method to create IOU/ expense report or the transaction thread, I think we should add it there too

Reply:
So, just to clarify: 
1. `addActions` is being used only in **Report.js** file by `addAttachment` and `addComment`.
<img width="905" alt="Screenshot 2023-10-19 at 16 19 49" src="https://github.com/Expensify/App/assets/57314631/0217b319-6939-4b42-aa0c-fdd016ef6abe">

2. Both functions used as independent in **ReportScreen.js** and **ReportActionCompose.js**. I don't see any connections with **IOU.requestMoney**.

What did I miss?

Reply:
We are also updating the behaviour of `buildOptimisticExpenseReport ` and others and that is used in requestMoney

Reply:
@mountiny, just covered all cases. Please take a look.